release/charts/charts.go

```diff
@@ -0,0 +1,61 @@
+package charts
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	ecmConfig "github.com/rancher/ecm-distro-tools/cmd/release/config"
+)
+
+// ListLifecycleStatus prints the lifecycle status of the charts
+func ListLifecycleStatus(ctx context.Context, c *ecmConfig.ChartsRelease, branch, chart string) error {
+
+	var branchArg, chartArg string = "", ""
+
+	branchArg = "--branch-version=" + branch
+	if chart != "" {
+		chartArg = "--chart=" + chart
+	}
+
+	err := executeChartsBuildScripts(c.Workspace, "lifecycle-status", branchArg, chartArg)
+	if err != nil {
```

Comment:
Let's rename this to: runChartsBuild. The previous name leaks implementation detail that could change in the future. This new name still conveys the idea and allows us to change from scripts to programs and exec model.

Reply:
Not sure if this isn't updating but it doesn't appear as though this has been completed.